spec/models/news_spec.rb

```diff
@@ -99,6 +99,8 @@
 
       user = FactoryGirl.create(:user)
       become_member_with_permissions(project, user)
+      # reload
+      project.members(true)
```

Comment:
Nice to see we have a helper called `BecomeMember`.

But isn't the much cooler way to do `FactoryGirl.create(:user, member_in_project: project, member_through_role: role)`?

In the long term we probably should only be using one way... I thought the one I am proposing is the one that you proposed to me earlier in my life ^^